src/storage/redis_metadata.h

```diff
@@ -94,6 +95,12 @@ enum RedisCommand {
 const std::vector<std::string> RedisTypeNames = {"none",   "string",    "hash",   "list",      "set",      "zset",
                                                  "bitmap", "sortedint", "stream", "MBbloom--", "ReJSON-RL"};
 
+const std::map<std::string, RedisType> RedisNamesMapType = {
+    {"none", kRedisNone},     {"string", kRedisString}, {"hash", kRedisHash},
+    {"list", kRedisList},     {"set", kRedisSet},       {"zset", kRedisZSet},
+    {"bitmap", kRedisBitmap}, {"stream", kRedisStream}, {"MBbloom--", kRedisBloomFilter},
+    {"ReJSON-RL", kRedisJson}};
+
```

Comment:
Please remove this.

Reply:
Thank you for your suggestion. I originally intended this design to reduce time complexity, but do you think it's not necessary to have too few array elements

Reply:
Yeah, the maintenance cost will be far more than the performance benefit we get.

Reply:
I understand. Thank you for your answer. I also missed an unverified type of test case. I will run the test case on the local branch first, and once it passes, please take a look on your end